### PR TITLE
Temporarily disabled check sending

### PR DESCRIPTION
**Pull request by iamcpdev**

### CODE REVIEW

**Review by maxwofford — APPROVED** [review]
The preview build works for me

**Comment by maxwofford** [comment]
Merge when you're ready @iamcpdev

File: app/views/checks/start_void.html.erb [hunk 1]
```diff
@@ -3,28 +3,10 @@
 
 <h1>Void a check to <%= @check.lob_address.name %></h1>
 
-<div class="card border b--warning pb0 mt3 mb3">
```
Comment (lachlanjc) [inline app/views/checks/start_void.html.erb:6]: Can we comment this out instead so it’s easy to put back when checks are back?

File: app/views/checks/_suspended.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  <div class="flex items-center warning">
+    <%= inline_icon 'important', size: 32, class: 'mr1' %>
+    <p class="bold mt0 mb0">Check sending and voiding is temporarily offline</p>
+  </div>
+  <p class="mt1 mb2">
+    <strong>Contact us if you need to void a check</strong>—your point of contact can help.
+    We're working with our check provider to fix this and will update with a specific date once we know more.
+  </p>
+  <div class="border-top card__banner card__darker slate">
+    <strong class="warning">Questions?</strong> Talk to your point of contact, <%= @event.point_of_contact.name %>
```
Comment (lachlanjc) [inline app/views/checks/_suspended.html.erb:11]: Let’s link the name to email as we do in other places

File: app/views/checks/_suspended.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<div class="card border b--warning pb0 mt3 mb3">
+  <div class="flex items-center warning">
+    <%= inline_icon 'important', size: 32, class: 'mr1' %>
```
Comment (lachlanjc) [inline app/views/checks/_suspended.html.erb:3]: Might add some personality to use `sad` icon here :)

File: app/views/events/transfers.html.erb [hunk 4]
```diff
@@ -2,14 +2,19 @@
...
 </h1>
 
 <section class="flex items-center justify-center flex-wrap pb2 md-pt2 md-pb3">
   <strong class="h3 muted md-pr2">Send a</strong>
   <div class="btn-group center">
-    <%= link_to 'Check', new_event_check_path(@event), class: 'btn bg-success' %>
+    <%= link_to 'Send a check', 
+        nil, #new_event_check_path(@event) 
+        class: 'btn bg-accent mr1', 
+        disabled: true %>
```
Comment (lachlanjc) [inline app/views/events/transfers.html.erb:17]: This is a bit of an accessibility/validity issue to continue using an `a` tag (links can’t be disabled), but I don’t think it’s a big deal.

Remove `mr1` though—might cause weird spacing on mobile?

**Review by lachlanjc — COMMENTED** [review]
Looks good, nice work! A few small notes

File: app/views/checks/start_void.html.erb (hunk 1 above)
Comment (iamcpdev) [reply app/views/checks/start_void.html.erb:6]: Done!

File: app/views/events/transfers.html.erb (hunk 4 above)
Comment (iamcpdev) [reply app/views/events/transfers.html.erb:17]: Done!

File: app/views/checks/_suspended.html.erb (hunk 3 above)
Comment (iamcpdev) [reply app/views/checks/_suspended.html.erb:3]: Done!

File: app/views/checks/_suspended.html.erb (hunk 2 above)
Comment (iamcpdev) [reply app/views/checks/_suspended.html.erb:11]: Done

**Comment by iamcpdev** [comment]
#818 @lachlanjc

**Pull request merged**